layouts/partials/navigators/navbar-2.html

```diff
@@ -47,6 +47,9 @@
       {{ if .IsTranslated }}
         {{ partial "navigators/lang-selector-2.html" . }}
       {{ end }}
+      {{ if site.Params.darkTheme.enable }}
```

Comment:
Should we rename `darkTheme` to `darkMode` ?

Reply:
Renamed this to darkMode.